sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/NamedVIPEvaluationStage.java

```diff
@@ -21,9 +21,8 @@
     public NamedVIPEvaluationStage(
             NamedVIPSpec namedVIPSpec,
             String taskName,
-            Optional<String> resourceId,
-            boolean useDefaultExecutor) {
-        super(namedVIPSpec, taskName, resourceId, useDefaultExecutor);
+            Optional<String> resourceId) {
+        super(namedVIPSpec, taskName, resourceId);
```

Comment:
Looks like you'll need to rebase. I'm getting rid of these arguments, but that'll be in later today or tomorrow.